panther_cli/main.py

```diff
@@ -15,16 +15,18 @@
 '''
 
 import argparse
+import base64
 from datetime import datetime
-from collections import defaultdict
 import importlib.util
+from collections import defaultdict
 from importlib.abc import Loader
 import json
 import logging
 import os
 import shutil
 import sys
 from typing import Any, Callable, DefaultDict, Dict, Iterator, List, Tuple
+import boto3
```

Comment:
Third-party imports like this one should be in the last import group all by itself (just like we do for Go)